docs/source/credits.rst

```diff
@@ -58,15 +58,14 @@ If you are using BibTex you may want to use this example BibTex entry::
 
 Funding
 -------
-*pyndl* supported by the Alexander von Humboldt Professorship awarded to R.
-Harald Baayen, the ERC advanced Grant (no. 742545), and by the University of
-Tübingen.
+*pyndl* has been supported by the Alexander von Humboldt Professorship awarded
+to R.  Harald Baayen, the ERC advanced Grant WIDE (no. 742545), and the
```

Comment:
Being very pedantic: There is a whitespace to much after the R.

I am not sure whether `sphinx` removes this while building. If it does, I am fine with it